evm/v0.5/src/helpers.ts

```diff
@@ -185,7 +185,7 @@ export async function assertActionThrows(action: AsyncFunction) {
 
   const ERROR_MESSAGES = ['invalid opcode', 'revert']
   const hasErrored = ERROR_MESSAGES.reduce(
-    (prev, next) => prev || e!.message.includes(next),
+    (prev, next) => (prev || e?.message?.includes(next)) ?? true,
     false,
   )
```

Comment:
`reduce` feels like overkill here. Should be able to use `ERROR_MESSAGES.some(...)` to accomplish the same thing and be less verbose.

Reply:
That is indeed true, thanks.